### PR TITLE
feat: register status ens usernames

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: ui/app/AppLayouts/Profile/Sections/Ens/ENSRegistered.qml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        font.pixelSize: 14
+        anchors.left: parent.left
+        anchors.right: parent.right
+        horizontalAlignment: Text.AlignHCenter
+        wrapMode: Text.WordWrap
+    }
+
+    StyledText {
+        id: progress
+        text: qsTr("You can follow the progress in the Transaction History section of your wallet.").arg(ensUsername)
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/ENSRegistered.qml:72]: Is the `arg` necessary? I don't see a `%1`

File: ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml [hunk 2]
```diff
@@ -23,6 +22,35 @@ Item {
...
 
+    ModalPopup {
+        id: transactionDialog
+        title: qsTr("TODO: replace this for the transaction dialog")
+
+        Input {
+            id: passwd
+            placeholderText: "Password"
+            anchors.top: parent.textToCopy
+            anchors.topMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml:33]:
```suggestion
            anchors.topMargin: Style.current.bigPadding
```

File: ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml [hunk 3]
```diff
@@ -23,6 +22,35 @@ Item {
         font.pixelSize: 20
     }
 
+    ModalPopup {
+        id: transactionDialog
+        title: qsTr("TODO: replace this for the transaction dialog")
+
+        Input {
+            id: passwd
+            placeholderText: "Password"
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml:31]:
```suggestion
            placeholderText: qsTr("Password")
```

File: ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml [hunk 4]
```diff
@@ -23,6 +22,35 @@ Item {
...
+            anchors.top: parent.textToCopy
+            anchors.topMargin: 24
+            anchors.left: parent.left
+            anchors.leftMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml:35]:
```suggestion
            anchors.leftMargin: Style.current.bigPadding
```

File: src/status/libstatus/stickers.nim [hunk 5]
```diff
@@ -134,7 +134,7 @@ proc buyPack*(packId: Stuint[256], address: EthAddress, price: Stuint[256], pass
     buyToken = BuyToken(packId: packId, address: address, price: price)
     buyTxAbiEncoded = stickerMktContract.methods["buyToken"].encodeAbi(buyToken)
   let
-    approveAndCallObj = ApproveAndCall(to: stickerMktContract.address, value: price, data: DynamicBytes[136].fromHex(buyTxAbiEncoded))
+    approveAndCallObj = ApproveAndCall(to: stickerMktContract.address, value: price, data: DynamicBytes[132].fromHex(buyTxAbiEncoded))
```
[the remark this answers is not in the document]
Comment (richard-ramos) [reply src/status/libstatus/stickers.nim:137]: This is a good question. I'm not sure why DynamicBytes requires a size. @emizzle, is it possible to use DynamicBytes without having to specify a size? `ApproveAndCall` should be able to accept data of any size

File: ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml (hunk 4 above)
Comment (richard-ramos) [reply ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml:35]: This one is not really necessary. This input field is part of a ModalPopup that needs to be replaced for a proper transaction dialog. I created it just to have some way to input the password but it should not be used in the final version

File: ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml (hunk 3 above)
Comment (richard-ramos) [reply ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml:31]: See comment below

File: ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml (hunk 2 above)
Comment (richard-ramos) [reply ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml:33]: See comment below x 2

**Pull request merged**